pkg/sql/contention/event_store.go

```diff
@@ -274,12 +272,12 @@ func (s *eventStore) forEachEvent(
 }
 
 func (s *eventStore) getEventByBlockingTxnID(
-	txnID uuid.UUID,
+	hash uint64,
```

Comment:
This function needs renaming (by Blocking + Waiting + Timestamp hash ID)

Reply:
Renamed.